concepticondata/references/references.bib

```diff
@@ -1,3 +1,18 @@
+@Article{Hwang2021,
+  Author                              = {Hwang, Yu M. and Yoonhye, Na and Sung-Bom P.},
+  Title                               = {Development of a categorical naming test in Korean: Standardization and clinical application for patients with stroke},
+  Year                                = {2021},
+  Journal                             = {PLOS ONE},
+  Volume                              = {16},
+  Number                              = {2},
+  Doi                                 = {10.1371/journal.pone.0247118}
+}
+@Book{Yonsei1998,
+  Author                              = {Language Information Research Institute of Yonsei University},
+  Title                               = {Yonsei Korean Dictionary},
+  Year                                = {1998},
+  Journal                             = {Doosan Donga}
```

Comment:
If it's a book, the "journal" category does not apply. It needs "publisher" and "address" instead.